utils/crypto/bls/public.go

```diff
@@ -23,14 +23,15 @@ type (
 	AggregatePublicKey = blst.P1Aggregate
 )
 
-// PublicKeyToBytes returns the compressed big-endian format of the public key.
-func PublicKeyToBytes(pk *PublicKey) []byte {
+// PublicKeyToCompressedBytes returns the compressed big-endian format of the
+// public key.
+func PublicKeyToCompressedBytes(pk *PublicKey) []byte {
 	return pk.Compress()
 }
 
-// PublicKeyFromBytes parses the compressed big-endian format of the public key
-// into a public key.
-func PublicKeyFromBytes(pkBytes []byte) (*PublicKey, error) {
+// PublicKeyFromCompressedBytes returns the uncompressed big-endian format of
+// the public key.
```

Comment:
question, should this be "parses the uncompressed big-endian format of the public key into a public key"?

Reply:
yes